Sources/CMakeLists.txt

```diff
@@ -2,4 +2,6 @@ add_subdirectory(UUID)
 add_subdirectory(Foundation)
 add_subdirectory(FoundationNetworking)
 add_subdirectory(FoundationXML)
-add_subdirectory(Tools)
+if(BUILD_TOOLS)
```

Comment:
This is not a standard CMake flag. Can you create a proper `option` for this for something like `FOUNDATION_BUILD_TOOLS`? We can control the default for it to disable it by default on WASM targets if desired.

Reply:
The option is already declared here as `option` https://github.com/apple/swift-corelibs-foundation/blob/7f62c8c52aeed9a86b503fe7b662c48eda5d0f8a/CMakeLists.txt#L43 but I agree the option name should be prefixed by `FOUNDATION`